test/Object/rlexe.xml

```diff
@@ -436,4 +436,10 @@
       <baseline />
     </default>
   </test>
+  <test>
+    <default>
+      <files>assign.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
```

Comment:
Shouldn't we have -trace:ObjectCopy so that we know the optimization is kicking in?

Reply:
I had that for my testing, but traces in tests can get noisy so didn't put it here. I can though since this probably shouldn't change much